cli/module_generate.go

```diff
@@ -608,6 +664,8 @@ func getLatestSDKTag(c *cli.Context, language string) (string, error) {
 	var repo string
 	if language == python {
 		repo = "viam-python-sdk"
+	} else if language == golang {
+		repo = "rdk"
 	}
```

Comment:
can we make this a switch statement? it's a bit more idiomatic and more consistent with elsewhere in this file.

also, it might make sense to add a `default` case? probably will never be relevant but no harm in defensive design here.

Reply:
sounds good! I am going to make the default case python because there are a few other places in the code where python seems to have been made default...but let me know if you think differently